plugins/ltac/tactic_option.ml

```diff
@@ -13,15 +13,11 @@ open Pp
 
 let declare_tactic_option ?(default=Tacexpr.TacId []) name =
   let locality = Summary.ref false ~name:(name^"-locality") in
-  let default_tactic_expr : Tacexpr.glob_tactic_expr ref =
-    Summary.ref default ~name:(name^"-default-tacexpr")
-  in
   let default_tactic : Tacexpr.glob_tactic_expr ref =
-    Summary.ref !default_tactic_expr ~name:(name^"-default-tactic")
+    Summary.ref default ~name:(name^"-default-tactic")
```

Comment:
>How did this code happen???

Before c81254903e1e50a2305cd48ccfb673d9737afc48 eval_tactic was here instead of in `get`